datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/search/SearchResolver.java

```diff
@@ -45,17 +48,20 @@
 
     return CompletableFuture.supplyAsync(() -> {
       try {
-        log.debug("Executing search. entity type {}, query {}, filters: {}, start: {}, count: {}", input.getType(),
-            input.getQuery(), input.getFilters(), start, count);
+        log.debug("Executing search. entity type {}, query {}, filters: {}, start: {}, count: {}, sort: {}", input.getType(),
```

Comment:
Recently, we've migrated to using the SearchAcrossEntitiesResolver from the UI. Are you intending to use the API from the UI or from the API?